app/routes/meetings/components/MeetingEditor.tsx

```diff
@@ -252,7 +262,32 @@ const MeetingEditor = () => {
                 name="report"
                 label="Referat"
                 component={EditorField.Field}
+                key={formKey}
               />
+
+              <FormSpy subscription={{ values: true }}>
+                {({ values }) => {
+                  useEffect(() => {
+                    const storedReport = localStorage.getItem(
+                      `meeting-${meeting.id}-temp`
+                    );
+                    const reportValue = values.report ?? '';
+
+                    if (
+                      !storedReport ||
+                      storedReport.length < reportValue.length
+                    ) {
+                      localStorage.setItem(
+                        `meeting-${meeting.id}-temp`,
+                        reportValue
+                      );
+                    }
+                  }, [values.report]);
```

Comment:
Should the update not be debounced a little?

Reply:
I changed it a little now, but I'm not really sure how it should be debounced? It's still in the onChange of the formspy.